akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -660,6 +664,15 @@ private final class Sdk(
       healthCheck = () => SdkRunner.FutureDone)
   }
 
+  private def isDisabled(disabledComponents: Set[Class[_]])(componentDescriptor: spi.ComponentDescriptor): Boolean = {
+    val className = componentDescriptor.implementationName
+    if (disabledComponents.map(_.getName).contains(className)) {
+      logger.info("Ignoring component [{}] as it is disabled in the configuration", className)
```

Comment:
in the Setup ?

Reply:
Technically, this could be a Setup or Integration test configuration. Maybe I should log both options separately?

Reply:
probably not important to have that detail, might be enough with 
```
"Ignoring component [{}] as it is disabled
```

Reply:
done